libc/src/sys/socket/linux/recvfrom.cpp

```diff
@@ -23,17 +23,26 @@ namespace LIBC_NAMESPACE_DECL {
 
 LLVM_LIBC_FUNCTION(ssize_t, recvfrom,
                    (int sockfd, void *buf, size_t len, int flags,
-                    struct sockaddr *__restrict dest_addr,
```

Comment:
does it make sense to keep `struct` here for consistency with header decl?

Reply:
These files are in C++, so I think it's better to also remove `struct` in the `recvfrom.h`.

Reply:
I decided to remove the struct from both so it's consistent.